### DISCUSSION TITLE
Background task name

### DISCUSSION THREAD

**Discussion by vitalybrandes** [OrchardCMS/OrchardCore#8865] [Q&A] [answered]
Can i some how override the full name of the background task?
![image](https://user-images.githubusercontent.com/61165619/111120281-d3700800-8573-11eb-8d0c-fcccb30e9e96.png)
instead this something else, including "Module name"?
Like "GetInformationTask"

**Comment by vitalybrandes** [comment, marked as answer]
Anyone?

**Reply by jtkech** [reply]
No, this is the Type.FullName

We would need to change the UI so that you could input a display text that would be used if provided

As i remember there is already a description that you can update

But for now there is no way to change the displayed task name

Can be suggested through an issue as a fetature request / enhancement

**Reply by vitalybrandes** [reply]
thank you
